tests/run-tests-on-external-cluster.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -e
+
+: {$KUBEVIRTNAMESPACE:="openshift-cnv"}
+_kubectl=$(which kubectl)
+_oc=$(which oc)
+
+[[ -z ${_kubectl} && -z ${_oc} ]] && echo "OC or KUBECTL is not installed" && exit 1
+
+[ -z "${KUBECONFIG}" ] && echo "KUBECONFIG is not set" && exit 1
+
+rm -rf _out
+hack/dockerized hack/build-func-tests.sh
```

Comment:
This brings an issue on newer systems since it requires Docker and will not run with Podman, even with the compatibility mode. That makes this less portable. I'd advice building the tests binary using go directly. I think that depending only on Go would make this more portable.

Reply:
You are suggesting to replace this with:
```
mkdir -p "${KUBEVIRT_DIR}/_out"                                           
GOPROXY=off GOFLAGS=-mod=vendor go test -c "${KUBEVIRT_DIR}/tests" -o "${KUBEVIRT_DIR}/_out/tests//tests.test"
```

This may work, but it also means we need to require that the machine building/running the tests has some packages/tools installed. That probably ends up less portable.

If I understand correctly, the main problem is that `hack/dockerized` is not working well with other container runtimes (e.g. podman)?

Reply:
Yeah, I would rather have this work with Go than with Docker. It would not be perfect but it would have fewer runtime dependencies.

Correct, `dockerized` won't work with Podman, not even in the podman-docker compatibility mode.

Reply:
#4022 should hopefully allow this to run with podman.